src/components/MultipleAvatars.js

```diff
@@ -193,8 +193,8 @@ function MultipleAvatars(props) {
                         ))}
                         {avatars.length > props.maxAvatarsInRow && (
                             <Tooltip
-                                // We only want to cap tooltips to only the first 10 users or so since some reports have hundreds of users, causing performance to degrade.
-                                text={tooltipTexts.slice(3, 10).join(', ')}
+                                // We only want to cap tooltips to only the first 20 users or so since some reports have hundreds of users, causing performance to degrade.
+                                text={tooltipTexts.slice(avatarRows.length * props.maxAvatarsInRow - 1, 20).join(', ')}
```

Comment:
@aimane-chnaif 
Tooltip text has max lines 3 then show Ellipsis effect, so the old way slice(3, 10) display just 7 users and this is enough because the ellipsis effect will apply.

But now `avatarRows.length * props.maxAvatarsInRow - 1`  = 7, so slice(7, 10) it displays tooltip text for only maximum 3 users, which don't show ellipsis effect.

So I update it to 20, and we can instead use dynamic value.
```javascript
tooltipTexts.slice(avatarRows.length * props.maxAvatarsInRow - 1, avatarRows.length * props.maxAvatarsInRow + 10)
```